resources/services/elasticbeanstalk/gen.hcl

```diff
@@ -0,0 +1,37 @@
+service          = "aws"
+output_directory = "."
+add_generate     = true
+
+resource "aws" "elasticbeanstalk" "application_versions" {
```

Comment:
Added only the `application_versions` one. Not sure there's a reason to add others for existing code